docs/index.rst

```diff
@@ -25,30 +25,36 @@ To install Flocker, first choose which Cluster Manager you are using, or intend
 	    <div class="pod-boxout pod-boxout--orchestration pod-boxout--recommended">
 			<img src="_static/images/docker2x.png" alt="Docker logo"/>
 			<span>Docker Swarm, with Docker Compose<em>Try it Now</em></span>
-	        <a href="docker-integration/" class="button button--fast">Install or Learn</a>
+	        <a href="docker-integration/" class="button button--fast">Using Flocker with Docker</a>
 	    </div>
 
 	    <div class="pod-boxout pod-boxout--orchestration">
 			<img src="_static/images/kubernetes2x.png" alt="Kubernetes logo"/>
 			<span>Kubernetes</span>
-	        <a href="kubernetes-integration/" class="button">Install or Learn</a>
+	        <a href="kubernetes-integration/" class="button">Using Flocker with Kubernetes</a>
 	    </div>
 
 	    <div class="pod-boxout pod-boxout--orchestration">
 			<img src="_static/images/mesos2x.png" alt="mesos logo"/>
 			<span>Mesos</span>
-	        <a href="mesos-integration/" class="button">Install or Learn</a>
+	        <a href="mesos-integration/" class="button">Using Flocker with Mesos</a>
 	    </div>
 	</div>
 
-Alternatively, if you want to install Flocker without a specific Cluster Manager in mind, that is also possible:
+Using Flocker without a Cluster Manager
+=======================================
+
+If you want to install Flocker without a specific Cluster Manager in mind, that is also possible:
 
 .. raw:: html
 
-	 <div class="pod-boxout pod-boxout--minor pod-boxout--orchestration">
-		<span><img src="_static/images/icon-question2x.png" aria-hidden="true" alt=""/>&nbsp;Install Flocker without a Cluster Manager</span>
-        <a href="flocker-standalone/" class="button">Install or Learn</a>
-    </div>
+   <div class="pods-eq">
+	    <div class="pod-boxout pod-boxout--2up pod-boxout--short">
```

Comment:
I personally like the one on doc-dev more than the one at http://clusterhq-staging-docs.s3-website-us-east-1.amazonaws.com/homepage-CTAupdates-FLOC-4091/ (as far as width of the box and the like on a full laptop)

Maybe the new one looks a little better on mobile, but I'm not sure that we should be optimizing for that.